src/types/Server.ts

```diff
@@ -368,4 +368,12 @@ export enum ServerAction {
   HTTP_REQUEST = 'HttpRequest',
   HTTP_RESPONSE = 'HttpResponse',
   HTTP_ERROR = 'HttpError',
+
+  // RESTful API
+  SIGNIN = 'signin',
+  SIGNON = 'signon',
+  SIGNOUT = 'signout',
+  RESTful_PASSWORD_RESET = 'passwordreset',
```

Comment:
RESTful_PASSWORD_RESET => REST_PASSWORD_RESET or RESTFUL_PASSWORD_RESET
All in uppercase.

Reply:
Done.